src/replace-shared-styles.js

```diff
@@ -1,39 +1,25 @@
-export default function(libraryStyles, lookup, foreignLookup) {
+export default function(libraryStyles, lookup, library) {
   const map = {}
   libraryStyles.forEach(librarySharedStyle => {
     const currentSharedStyle = lookup[librarySharedStyle.name]
     if (currentSharedStyle) {
-      const imported = librarySharedStyle.import()
-      // if the shared style is coming from a library, then we just want to:
-      // - import the matching style
```

Comment:
any reason to remove comments instead of adding more? 😄

Reply:
mmm true, I will get them back ;)